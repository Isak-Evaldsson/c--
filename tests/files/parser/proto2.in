int test_proto_2(int a);