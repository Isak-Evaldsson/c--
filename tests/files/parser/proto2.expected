Prototype: test_proto_2
    Param: a
